### PR TITLE
Transaction History Status (NMA-20)

### PR DESCRIPTION
**Pull request by HashEngineering**
This should fail Travis CI checks because it depends on https://github.com/dashevo/dashj/pull/63 for changes in DashJ.

The **TransactionUtil** class was added to allow determination of status text to be reused in other parts of the app.  It will be useful on the Transaction Details Popup. 

This story depends on #263 for the "on click" action.

This should be merged after #263 is merged and this PR is rebased or merged against `redesign`.

### CODE REVIEW

File: wallet/src/de/schildbach/wallet/ui/TransactionsAdapter.java [hunk 1]
```diff
@@ -247,16 +238,20 @@ public void onClick(final View v) {
...
-                    public void onClick(final View v) {
-                        onClickListener.onTransactionMenuClick(v, tx);
-                    }
-                });
+           //     transactionHolder.menuView.setOnClickListener(new View.OnClickListener() {
+           //         @Override
+           //         public void onClick(final View v) {
+           //             onClickListener.onTransactionMenuClick(v, tx);
+           //         }
+           //     });
```
Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/ui/TransactionsAdapter.java]: these comments are left in for now.  This section of code will probably have merge conflicts with #263.

Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/ui/TransactionsAdapter.java]: these comments are left in for now. This section of code will probably have merge conflicts with #263.

**Comment by denlb** [comment]
Tests are failing

**Comment by HashEngineering** [comment]
The test will be passing within 2 hours.

**Comment by HashEngineering** [comment]
Test are not yet passing.  Perhaps the new dashj 0.17-SNAPSHOT was not available earlier.

**Comment by HashEngineering** [comment]
I may make a change to the Transaction Details Popup code so the app doesn't crash on internal transactions.

**Comment by HashEngineering** [comment]
#263 has been merged.

**Pull request merged**